### PR TITLE
Update all available dependencies

### PR DESCRIPTION
**Pull request by boswelja**
Also a minor rewrite to use `@googleapis/androidpublisher`, which contains only androidpublisher-related parts of googleapis. The result is a much smaller compiled library

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -28,17 +32,16 @@ async function run(): Promise<void> {
     core.exportVariable('GOOGLE_APPLICATION_CREDENTIALS', serviceAccountFile)
 
     core.debug('Authenticating: creating auth client')
-    const auth = new google.auth.GoogleAuth({
+    const authClient = new auth.GoogleAuth({
```
Comment (kevin-david) [inline src/main.ts:35]: this is never used - I guess `auth.GoogleAuth` manages this internally somehow and just passing `auth` to future methods "just works"?

**Review by kevin-david — APPROVED** [review]
Looks good, left one comment just to confirm something but feel free to resolve it & merge if no changes required.

File: src/main.ts (hunk 1 above)
Comment (boswelja) [reply src/main.ts:35]: I haven't verified it yet, but we get type mismatch issues if we pass in the result of getAuthClient, whereas this seems to be ok.

It's strange, considering the documentation for them doesn't reflect this either.

Comment (boswelja) [reply src/main.ts:35]: I double-checked the type def and it definitely is set up to accept a GoogleAuth directly. I say we merge it and hope for the best 😆

**Pull request merged**